GTG/gtk/ui/preferences.ui

```diff
@@ -0,0 +1,497 @@
+            <property name="visible">True</property>
+            <property name="expand">False</property>
+          </object>
+        </child>
+        <child>
+          <object class="GtkStack" id="stack">
+            <property name="visible">True</property>
+            <property name="expand">True</property>
+            <property name="visible-child">general_pref_window</property>
+            <property name="transition-type">GTK_STACK_TRANSITION_TYPE_CROSSFADE</property>
+            <child>
+              <object class="GtkScrolledWindow" id="general_pref_window">
+                <property name="min-content-height">500</property>
+                <property name="hscrollbar-policy">never</property>
+                <property name="visible">true</property>
+                <child>
+                  <object class="GtkBox">
+                    <property name="visible">True</property>
+                    <property name="orientation">vertical</property>
+                    <child>
+                      <object class="GtkLabel" id="system_label">
+                        <property name="visible">true</property>
+                        <property name="label" translatable="yes">System</property>
+                        <property name="xalign">0.0</property>
+                        <property name="margin-start">10</property>
+                        <property name="margin-top">6</property>
+                        <style>
+                          <class name="dim-label"/>
+                        </style>
+                        <attributes>
+                          <attribute name="weight" value="bold"/>
+                        </attributes>
+                      </object>
+                    </child>
+                    <child>
+                      <object class="GtkBox" id="autostart_box">
+                        <property name="margin-start">40</property>
+                        <property name="margin-end">20</property>
+                        <property name="margin-top">10</property>
+                        <property name="homogeneous">False</property>
+                        <child>
+                          <object class="GtkSwitch" id="autostart_button">
+                            <property name="visible">True</property>
+                            <property name="valign">center</property>
+                            <signal handler="on_autostart_button_toggled" name="state-set" swapped="no"/>
+                          </object>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="orientation">vertical</property>
+                            <property name="visible">true</property>
+                            <property name="margin-start">10</property>
+                            <child>
+                              <object class="GtkLabel" id="autostart_title">
```

Comment:
personal preference (I'm the one who's scrolling through the code all day :smile: ), but if you insist (I see your point), I'll tidy up this thing